qa/tasks/cephfs/test_volumes.py

```diff
@@ -57,13 +60,33 @@ def _get_clone_status(self, clone, clone_group=None):
         result = json.loads(self._fs_cmd(*args))
         return result
 
-    def _wait_for_clone_to_complete(self, clone, clone_group=None, timo=120):
+    def _wait_for_clone_to_be_pending(self, clone, clone_group=None,
+                                      timo=120, sleep=1):
+        # check for "in-progress" state too along with "pending" state, because
+        # if former has occurred it means latter has occured before (which can
+        # happen for such a small time that it is easy to miss) and it won't
+        # occur again.
+        states = ('pending', 'in-progress')
+        self.__check_clone_state(states, clone, clone_group, timo, sleep)
```

Comment:
`__check_clone_state` does not accept `sleep` argument anymore.

Reply:
Fixed.